lib/qml/lib/demo.py

```diff
@@ -152,6 +153,7 @@ def build(
     quiet: bool = False,
     keep_going: bool = False,
     dev: bool = False,
+    venv: Optional[str] = None,
```

Comment:
Consistent with Python 3.11+,
```suggestion
    venv: str | None = None,
```